### PR TITLE
CW-695: Fix Zone Mismatch Error Triggered By Restoring Backup

### PR DESCRIPTION
**Pull request by Blazebrain**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [ ] Initial Manual Tests Passed
- [ ] Double check modified code and verify it with the feature/task requirements
- [ ] Format code
- [ ] Look for code duplication
- [ ] Clear naming for variables and methods

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -62,49 +65,27 @@ Future<void> main() async {
...
+        TopLevelErrorWidget(error: error, stackTrace: stackTrace),
       );
     }
 
     ExceptionHandler.onError(FlutterErrorDetails(exception: error, stack: stackTrace));
   });
 }
 
+Future<void> initializeAppAtRoot({bool reInitializing = false}) async {
+  if (reInitializing) await setDefaultMinimumWindowSize();
```
Comment (OmarHatem28) [inline lib/main.dart:84]: this was set when app is initialized normally, why change it to be only when restore?

**Review by OmarHatem28 — COMMENTED** [review]
let's test this thoroughly tho

File: lib/main.dart (hunk 1 above)
Comment (Blazebrain) [reply lib/main.dart:84]: Ohh, error from my end.

It's supposed to be inverse. I'll adjust it now.

The check is, if it's not a restore flow, don't call the defaults again.

Comment (Blazebrain) [reply lib/main.dart]: Done

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**